baseplate/lib/message_queue.py

```diff
@@ -140,6 +262,12 @@ def queue_tool() -> None:
         help="if creating the queue, what to set the maximum message size to",
     )
     parser.add_argument("queue_name", help="the name of the queue to consume")
+    parser.add_argument(
+        "--queue_type",
+        default=QueueType.POSIX.value,
+        choices=[qt.value for qt in QueueType],
+        help="whether to use an in-memory queue or a posix queue",
```

Comment:
The queue options will be listed automatically but the Argparse library so you don;t need to state them here. I would just say something generic like: "Allows selection of the queue implementation"